.changes/unreleased/tinymce-TINY-11215-2025-01-10.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Fixed
+body: Table resize bars now remain properly aligned when the inline editor is used inside a scrollable container.
```

I think we tend to use past tense so something like:

```suggestion
body: Table resize bars was not properly aligned for inline editors inside scrollable containers.
```